frame/support/test/src/lib.rs

```diff
@@ -41,3 +41,15 @@ frame_support::decl_module! {
 	/// Some test module
 	pub struct Module<T: Config> for enum Call where origin: T::Origin, system=self {}
 }
+
+/// A PalletInfo implementation which just panics.
+pub struct PanicPalletInfo;
+
+impl frame_support::traits::PalletInfo for PanicPalletInfo{
+	fn index<P: 'static>() -> Option<usize> {
+		unimplemented!("PanicPalletInfo mustn't be triggered by tests");
+	}
+	fn name<P: 'static>() -> Option<&'static str> {
+		unimplemented!("PanicPalletInfo mustn't be triggered by tests");
+	}
+}
```

Comment:
Why don't you use it from `frame-support`?

Reply:
in frame-support I didn't expose it, it is hidden behind the `#[cfg(test)]` flag, but we could make it public.

Reply:
Ahh I had overseen this. Maybe move it at least from the `traits` file then?

Reply:
yes, I moved it to `tests` module